core/src/snapshot.rs

```diff
@@ -0,0 +1,181 @@
+//! This module contains [`WorldStateView`] snapshot actor service.
+#![allow(
+    clippy::std_instead_of_core,
+    clippy::std_instead_of_alloc,
+    clippy::arithmetic_side_effects
+)]
+use std::{io::Read, path::Path, sync::Arc, time::Duration};
```

Comment:
```suggestion
)]

use std::{io::Read, path::Path, sync::Arc, time::Duration};
```

More lines to the god of lines!